packages/@webex/plugin-cc/src/services/task/TaskManager.ts

```diff
@@ -72,16 +72,20 @@ export default class TaskManager extends EventEmitter {
             this.currentTask = this.currentTask.updateTaskData(payload.data);
             this.currentTask.emit(TASK_EVENTS.TASK_ASSIGNED, this.currentTask);
             break;
+          case CC_EVENTS.AGENT_CONTACT_OFFER_RONA:
           case CC_EVENTS.CONTACT_ENDED:
-            this.emit(TASK_EVENTS.TASK_UNASSIGNED, this.currentTask);
-            if (this.webCallingService.loginOption === LoginOption.BROWSER) {
-              this.currentTask.unregisterWebCallListeners();
-              this.webCallingService.unregisterCallListeners();
+            if (this.currentTask.data.interaction.state === 'new') {
```

Comment:
question. Will we be adding other states in the future? We could make `new` a constant.

Reply:
We may or may not